### PR TITLE
feat: add docs for backup to the cloud

### PR DESCRIPTION
**Pull request by dragonly**
Close BYT-1332.

### CODE REVIEW

File: content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+docker run --init \
+  --name bytebase \
+  --restart always \
+  --publish 5678:8080 \
+  --health-cmd "curl --fail http://localhost:5678/healthz || exit 1" \
+  --health-interval 5m \
+  --health-timeout 60s \
+  --volume ~/.bytebase/data:/var/opt/bytebase \
+  --volume ~/.aws:/var/opt/bytebase/.aws \
+  bytebase/bytebase:1.4.0 \
```
Comment (tianzhou) [inline content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md:91]: use %%bb_version%%

File: content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  --backup-region us-east-1 \
+  --backup-bucket s3://bytebase-dev \
+  --backup-credential /var/opt/bytebase/.aws/credentials
+```
+
+Now Bytebase will start with all backup files stored in the cloud. All operations for backup and restore are the same as before.
+
+<hint-block type="warning">
+
+Caveat: If you used to run Bytebase without these flags, you might already have backups in your local disk. They can be successfully restored even if you run Bytebase with the new flags to store new backups in the AWS S3. But we still recommend that you do at least one backup for the production databases after the new Bytebase deployment. This ensure that there are backups in the cloud storage and that your data is safe.
```
Comment (tianzhou) [inline content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md:103]:
```suggestion
Caveat: If you used to run Bytebase without these flags, you might already have backups in your local disk. They can be successfully restored even if you run Bytebase with the new flags to store new backups in the AWS S3. But we still recommend that you do at least one backup for the production databases after Bytebase switches to the cloud backup. This ensures that there are backups in the cloud storage and that your data is safe.
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md (hunk 1 above)
Comment (dragonly) [reply content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md]: Fixed.

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            "Resource": [
+                "arn:aws:s3:::<YOUR-BUCKET-NAME-HERE>",
+                "arn:aws:s3:::<YOUR-BUCKET-NAME-HERE>/*"
+            ]
+        }
+    ]
+}
+```
+
+Please substitute the `<YOUR-BUCKET-NAME-HERE>` part with your actual bucket name. Do not change the Version field’s value, which is the newest version of the AWS policy language.
```
Comment (milasuperstar) [inline content/docs/en/disaster-recovery/backup-restore-database/cloud-backup.md:44]:
```suggestion
Please substitute the `<YOUR-BUCKET-NAME-HERE>` part with your actual bucket name. Do not change the "Version" field’s value, which is the newest version of the AWS policy language.
```

**Review by milasuperstar — COMMENTED** [review]
rest LGTM

**Pull request merged**